mewarpx/mewarpx/diags_store/field_diagnostic.py

```diff
@@ -118,6 +118,9 @@ def fields_diag(self):
 
         if self.process_phi:
             data = mwxrun.get_gathered_phi_grid(include_ghosts=False)
+            if mwxrun.dim == 1:
+                data = np.tile(data, mwxrun.nz//2).reshape(
+                    mwxrun.nz//2, data.shape[0])
```

Comment:
Is there a comment about why the `//2` is here?

Reply:
It is completely arbitrary just to give the 2d plot a finite width, so not physical at all.

Reply:
I changed this to only tile the data being plotted but not the array that is saved.